### DISCUSSION TITLE
Failed to resolve: org.mobilenativefoundation.store:core:5.1.0-alpha01

### DISCUSSION THREAD

**Discussion by har-nick** [dropbox/Store#599] [Q&A] [answered]
Hi there! I'd like to try out the latest alpha to use the converter, however I can't seem to resolve it with Gradle.

I've checked mvnrepository and found the files [are available](https://repo1.maven.org/maven2/org/mobilenativefoundation/store/store5/5.1.0-alpha01/), but switching the version from `5.0.0` to `5.1.0-alpha01` causes it to fail.

#### libs.version.toml
```
store = { module = "org.mobilenativefoundation.store:store5", version = "5.1.0-alpha01" }
```

Is there something I've done wrong?

**Comment by matt-ramotar** [comment]
Hey there, can you share your build.gradle and I'll take a look?

**Comment by matt-ramotar** [comment]
Ah, it looks like `core` wasn't released. Will release asap

**Comment by har-nick** [comment]
Thanks for the quick response!

**Comment by matt-ramotar** [comment]
#600

**Comment by matt-ramotar** [comment, marked as answer]
Hey @har-nick - Please try with 5.1.0-alpha02
